Cargo.lock

```diff
@@ -5256,9 +5256,9 @@ checksum = "dc375e1527247fe1a97d8b7156678dfe7c1af2fc075c9a4db3690ecd2a148068"
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.59"
+version = "1.0.63"
```

Comment:
This fixes an issue with newer nightlies, but I couldn't jump to latest yet because swc_ecma_ast has an issue with it (already fixed, just need to update SWC here).